tables/top_level_geographies.md

```diff
@@ -7,11 +7,11 @@ The following `JOIN` queries can be carried out:
 - `geography_type_id` on the [geography_groupings](geography_groupings.md) table using `geography_type_id`.
 - `top_level_geography_id` on the [geography_areas](geography_areas.md) table using `id`.
 
-## What are top-level geographies?
+## What Are top-level geographies?
```

Comment:
I reckon for all headers we should be using title case:

https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case

So in this case (I think) it would be: 

## What are Top-Level Geographies?

Reply:
thanks for the feedack, I agree have reverted thiose title changes